conda/testing/fixtures.py

```diff
@@ -468,6 +483,16 @@ def tmp_env(
     yield TmpEnvFixture(path_factory, conda_cli)
 
 
+@pytest.fixture
+def empty_env(tmp_env: TmpEnvFixture) -> Path:
+    """A function scoped fixture returning an empty environment.
+
+    Use this when creating a conda environment that is empty.
+    """
+    with tmp_env() as prefix:
```

Comment:
```suggestion
    with tmp_env(shallow=True) as prefix:
```

I think this may make it more obvious that empty envs are shallow by default?